class.jetpack.php

```diff
@@ -1809,9 +1809,43 @@ function user_role_change( $user_id ) {
 	}
 
 	/**
-	 * Loads the currently active modules.
+	 * Loads the private module if it has been activated.
 	 */
-	public static function load_modules() {
+	public static function load_private() {
```

Comment:
>  Is this a left-over from before? 

Nope, this is a new change introduced for this module. More details below.

> Why do we need this when other modules don't? 

Giving an example of an issue I spotted. Say Sitemap module is activated first, and this is followed by activating the Private module, then even though Sitemap is "deactivated" by `jetpack_active_modules` filter but `http://example.org/sitemap.xml` still renders the sitemap. 
I believe this is because the Sitemap module is loaded before Private module. One way to solve this would be to make Sitemaps load itself after all modules are loaded:

In _modules/sitemaps.php_:
```
add_action( 'plugins_loaded', 'jetpack_sitemap_load', 105 ); // After Jetpack::load_modules()

function jetpack_sitemap_load() {
	if ( '1' == get_option( 'blog_public' ) ) { // loose comparison okay.
		include_once 'sitemaps/sitemaps.php';
}
```

But this only specifically solves the problem for Sitemaps. I think having the Private module always load first, before any other module, would ensure that the `jetpack_active_modules` filter among others would always be respected by every other module. So, this fixes the issue of not rendering `http://example.org/sitemap.xml` when Private module is activated. 

> add_private_dashboard_glance_items is already in modules/private/class-jetpack-private.php, why isn't add_public_dashboard_glance_items and its hook in there as well, so it gets loaded when the module is activated?

I wanted `add_public_dashboard_glance_items()` to be called when the private module is not activated. This shows the "...site is public" message here:

<img width="482" alt="Screenshot 2019-05-21 at 5 44 03 PM" src="https://user-images.githubusercontent.com/1269602/58095189-11a67580-7bf0-11e9-89fd-5917e70e03db.png">

Reply:
Ok, that makes sense. Thank you!